share/drom/skeletons/packages/menhir_lib/dune_

```diff
@@ -0,0 +1,20 @@
+![file:dune]; generated by drom from package skeleton '!{skeleton}'
+
+(library
+  (public_name !{name})
+  (wrapped !{pack-modules})
+  (libraries !{dune-libraries} !(package-dune-libraries))
+  !{dune-stanzas}
+  !(package-dune-stanzas)
+  )
+
+!{package-dune-files}
+(documentation
+  (package !{package-name}))
+!(package-dune-trailer)
+
+(ocamllex
```

Comment:
These lines are automatically added by `drom` if the line `generators = [ "ocamllex", "menhir" ]` is in the `package.toml` file.

Reply:
I added a `package.toml` file with the corresponding line and it doesn't seems to work.